cmd/vpd/src/lib.rs

```diff
@@ -201,28 +201,38 @@ fn list(
 ) -> Result<()> {
     let devices = vpd_devices(hubris).collect::<Vec<_>>();
     let mut context = HiffyContext::new(hubris, core, subargs.timeout)?;
-    let op = hubris.get_idol_command("Vpd.is_locked")?;
     let read_op = hubris.get_idol_command("Vpd.read")?;
 
-    let mut ops = vec![];
+    let locked_op = hubris.get_idol_command("Vpd.is_locked").ok();
+    let results = if let Some(locked_op) = &locked_op {
+        let mut ops = vec![];
 
-    for ndx in 0..devices.len() {
-        let payload =
-            op.payload(&[("index", idol::IdolArgument::Scalar(ndx as u64))])?;
+        for ndx in 0..devices.len() {
+            let payload = locked_op.payload(&[(
+                "index",
+                idol::IdolArgument::Scalar(ndx as u64),
+            )])?;
 
-        context.idol_call_ops(&op, &payload, &mut ops)?;
-    }
+            context.idol_call_ops(locked_op, &payload, &mut ops)?;
+        }
 
-    ops.push(Op::Done);
+        ops.push(Op::Done);
 
-    let results = context.run(core, ops.as_slice(), None)?;
+        Some(context.run(core, ops.as_slice(), None)?)
+    } else {
+        println!(
+            "Note: firmware does not support the Vpd.is_locked operation."
+        );
+        println!("Lock status will be missing from the table below.");
```

Comment:
should these maybe go to stderr rather than stdout? maybe they should use the `humility_log::warn!` macro?

Reply:
I wanted to make sure they appeared right next to the table, if the table was e.g. piped into a file or copy-pasted. Otherwise stderr would have been great... I was just worried they wouldn't get seen and then someone would be like "what is this too old stuff"

I can be talked out of that.

Reply:
makes sense to me!